framework/test/unit/modules/chain/unit/modules/blocks/process.js

```diff
@@ -1729,132 +1730,76 @@ describe('blocks/process', async () => {
-								'accounts.getAccount-ERR',
-								null
-							)
+							modules.blocks.process.processTransactions.verifyTransactions.throws()
 						);
 
 						it('should call a callback with error', done => {
 							blocksProcessModule.generateBlock(
 								{ publicKey: '123abc', privateKey: 'aaa' },
 								41287231,
 								err => {
-									expect(err).to.equal('Sender not found');
+									expect(err).to.equal('Transaction type not found.');
 									done();
 								}
 							);
 						});
 					});
 
 					describe('when succeeds', async () => {
-						beforeEach(() =>
-							modules.accounts.getAccount.callsArgWith(1, null, true)
-						);
-						afterEach(
-							async () =>
-								expect(modules.blocks.verify.processBlock.calledOnce).to.be.true
-						);
-
-						describe('library.logic.transaction.ready', async () => {
-							describe('when returns false', async () => {
-								beforeEach(() =>
-									library.logic.transaction.ready.returns(false)
-								);
-
-								it('should generate block without transactions', done => {
-									blocksProcessModule.generateBlock(
-										{ publicKey: '123abc', privateKey: 'aaa' },
-										41287231,
-										err => {
-											expect(err).to.be.null;
-											expect(library.logic.transaction.verify.calledOnce).to.be
-												.false;
-											expect(
-												modules.blocks.verify.processBlock.args[0][0]
-													.transactions.length
-											).to.equal(0);
-											done();
-										}
-									);
-								});
-							});
-
-							describe('when returns true', async () => {
-								beforeEach(() => library.logic.transaction.ready.returns(true));
-
-								describe('library.logic.transaction.verify', async () => {
-									describe('when fails', async () => {
-										beforeEach(() =>
-											library.logic.transaction.verify.callsArgWith(
-												4,
-												'transaction.verify-ERR',
-												null
-											)
-										);
+						beforeEach(async () => {
+							modules.blocks.process.processTransactions.verifyTransactions.returns({ id: 1, status: 1, errors: [] }, { id: 2, status: 1, errors: [] });
+						});
 
-										it('should generate block without transactions', done => {
-											blocksProcessModule.generateBlock(
-												{ publicKey: '123abc', privateKey: 'aaa' },
-												41287231,
-												err => {
-													expect(err).to.be.null;
-													expect(
-														modules.blocks.verify.processBlock.args[0][0]
-															.transactions.length
-													).to.equal(0);
-													done();
-												}
-											);
-										});
-									});
+						it('should generate block with transactions', done => {
+							blocksProcessModule.generateBlock(
+								{ publicKey: '123abc', privateKey: 'aaa' },
+								41287231,
+								err => {
+									expect(err).to.be.null;
+									expect(
+										modules.blocks.verify.processBlock.args[0][0]
+											.transactions.length
```

Comment:
Could it be possible to also test here that the included transactions are the correct ones?